src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -583,8 +583,8 @@ public string MarkdownFilesDirectory
         /// <summary>
         /// CustomNodeDefinitions property 
         /// </summary>
-        private List<CustomNodeDefinition> customNodeDefinitions;
-        public List<CustomNodeDefinition> CustomNodeDefinitions
+        private ObservableCollection<CustomNodeDefinition> customNodeDefinitions = new ObservableCollection<CustomNodeDefinition>();
+        public ObservableCollection<CustomNodeDefinition> CustomNodeDefinitions
```

Comment:
changing a public signature to return a different type is an API breaking change.

Reply:
Good point, will revert and try another approach!